arches_search/migrations/0008_termsearch_search_vector_and_more.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 5.2.3 on 2025-12-10 12:10
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "arches_search",
+            "0007_alter_booleansearch_resourceinstanceid_and_more",
+        ),
+        ("models", "12394_resource_identifier"),
+    ]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name="termsearch",
+            name="arches_sear_search__40996d_gin",
+        ),
+        migrations.RemoveField(
+            model_name="termsearch",
+            name="search_vector",
+        ),
+        migrations.AddField(
+            model_name="termsearch",
+            name="search_vector",
+            field=models.GeneratedField(
+                db_persist=True,
+                expression=django.contrib.postgres.search.SearchVector(
+                    "value", config="english"
+                ),
+                null=True,
+                output_field=django.contrib.postgres.search.SearchVectorField(),
+            ),
+        ),
+        migrations.AddIndex(
+            model_name="termsearch",
+            index=django.contrib.postgres.indexes.GinIndex(
+                fields=["search_vector"], name="arches_sear_search__40996d_gin"
+            ),
+        ),
```

Comment:
Django wouldn't allow an 'AlterField' when switching to a generated column so I had to drop the field and index and re-add them.